openshift/release/knative-eventing-kafka-broker-dp-ci.yaml

```diff
@@ -403,3 +403,244 @@ spec:
       protocol: TCP
       targetPort: 9090
 ---
+---
```

Comment:
```suggestion
```

Reply:
these are autogenerated 
the generator is not yet fully in place

so I ignore them for now